.github/workflows/circleci.yml

```diff
@@ -584,9 +598,10 @@ jobs:
     timeout-minutes: 60
     needs: [install-dependencies, lint-checks, contractkit-tests]
     if: |
+      github.base_ref == 'master' || contains(github.base_ref, 'staging') || contains(github.base_ref, 'production') ||
       contains(needs.install-dependencies.outputs.all_modified_files, 'packages/protocol') ||
```

Comment:
This should also run when changes are made to:
- `@celo/phone-number-privacy-common`
- `@celo/dev-utils`
- `@celo/typescript`